website/content/blog/2024-9-30-graphite-progress-report-q3-2024.md

```diff
@@ -1,24 +1,39 @@
 +++
 title = "Graphite progress report (Q3 2024)"
-date = 2024-09-30
+date = 2024-10-02
 [extra]
-banner = "ToDo"
-banner_png = "ToDo"
-author = "Keavon Chambers"
-summary = "Graphite's Q3 2024 update introduces node graph organisation, non destructive vector editing, and faster boolean operations."
-reddit = "ToDo"
-twitter = "ToDo"
+banner = "https://files.keavon.com/-/OddUnfortunateLeopard/capture.png"
+banner_png = "https://files.keavon.com/-/OddUnfortunateLeopard/capture.png"
+author = "Keavon Chambers & Hypercube"
+summary = "TODO: Summary"
+reddit = "TODO: Reddit link"
+twitter = "TODO: Twitter link"
 +++
 
-[Graphite](/), an open source 2D procedural graphics editor, has spent the months of July, August, and September introducing node graph organisation, non destructive vector editing, faster boolean operations, and many more improvements.
+[Graphite](/), a new open source 2D procedural graphics editor, has spent April–June introducing **boolean path operations, a new gradient picker, layer locking**, and more improvements.
 
-In this quater we have continued iterating on the usability of our unqiue node powered vector workflow. As we reach the conclusion of the [Google Summer of Code](/blog/graphite-internships-announcing-participation-in-gsoc-2024/) projects, we now have a polished node graph organisation system, a working RAW image pipeline (not yet in the editor), and a WebGPU based renderer (File -> Preferences -> Vello). Thanks to the students for their hard work on this and to Google's open source team for sponsoring.
+Overall, editor functionality has been shaping up and becoming an all around useful tool suite, with notable reductions in rough edges for the vector graphics workflow (our initial focus). Raster and raw photo processing workflows are also now in-development by our [Google Summer of Code](/blog/graphite-internships-announcing-participation-in-gsoc-2024/) student interns. Node graph quality-of-life improvements centered around tidy node organization are also the focus of the summer work that's underway. These projects are detailed below.
```

Comment:
I assume I shouldn't bother to write any prose in future (given that you don't even want it as a starting point)?